src/Drawer/Drawer.js

```diff
@@ -28,42 +29,42 @@ export const styleSheet = createStyleSheet('Drawer', (theme) => {
   };
 });
 
+type Props = {
+  /**
+   * The contents of the `Drawer`
+   */
+  children?: Object,
+  /**
+   * The CSS class name of the root element.
+   */
+  className?: string,
+  /**
+   * If set to true, the drawer will dock itself
+   * and will no longer slide in with an overlay
+   */
+  docked?: boolean,
+  open: boolean,
+  /**
+   * The CSS class name of the paper element.
+   */
+  paperClassName?: string,
+  zDepth: number,
```

Comment:
Without a `?`, it is required.  I'm still working these, but you can see below that a default value is provided.